profiles/common/modules/custom/nexteuropa_token/nexteuropa_token.api.php

```diff
@@ -25,3 +25,12 @@ function hook_nexteuropa_token_token_handlers() {
 function hook_nexteuropa_token_token_handlers_alter(array &$handlers) {
   $handlers['handler_name'] = '\Drupal\module_name\HandlerNameTokenHandler';
 }
+
+/**
+ * Implements hook_nexteuropa_token_entity_view_mode_type().
+ */
+function hook_nexteuropa_token_entity_view_mode_type() {
+  return array(
+    'handler_name' => '\Drupal\module_name\HandlerName',
```

Comment:
instead of `handler_name` we should have `entity_type` since we require it is the same as the entity type machine name right?